modules/targets/darwin/keybindings.nix

```
@@ -0,0 +1,43 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  inherit (pkgs.stdenv.hostPlatform) isDarwin;
+  cfg = config.targets.darwin;
+  homeDir = config.home.homeDirectory;
+  confFile = pkgs.writeText "DefaultKeybinding.dict"
+    (lib.generators.toPlist { } cfg.keybindings);
+in {
+  options.targets.darwin.keybindings = mkOption {
+    type = types.attrs;
```

```suggestion
    type = types.attrsOf types.str;
```

Unless it needs to be unevaluated, use `types.attrsOf` which allows for declaration merging (`mkIf`, etc.).

---

Fixed. It's now a `attrsOf anything` though because multi-key bindings can be defined using nested attrs.